pkg/apis/serving/v1alpha1/llm_inference_service_types.go

```diff
@@ -235,10 +248,6 @@ type SchedulerSpec struct {
 // InferencePoolSpec defines the configuration for an InferencePool.
 // 'Spec' and 'Ref' are mutually exclusive.
 type InferencePoolSpec struct {
-	// Spec defines an inline InferencePool specification.
-	// +optional
-	Spec *igwapi.InferencePoolSpec `json:"spec,omitempty"`
```

Comment:
This is a breaking API change, we need to keep v1alpha1 the same API structure as before in order to upgrade safely